composer.json

```diff
@@ -47,7 +46,7 @@
 	},
 	"extra": {
 		"branch-alias": {
-			"dev-master": "1.5.x-dev"
+			"dev-master": "1.6.x-dev"
```

Comment:
For reference: This was manually fixed without a pull request in b9c6c7adbdd5c00609b74cdc36ed52900536e091 and 64a23319d75c9cc2363ba7654ac83fdfb7d6466d.